src/ActionList/shared.ts

```diff
@@ -52,18 +58,21 @@ type MenuItemProps = {
   'aria-labelledby'?: string
   'aria-describedby'?: string
   role?: string
+  'data-inactive'?: string
```

Comment:
I don't think this should be a prop. We set this based on `inactiveText` value right?